### ISSUE TITLE
Random bad performance in and out of full screen

### ISSUE THREAD

**Issue by WizLizard** [punesemu/puNES#51]
I've only tested this with the latest 64bit d3d9 WIP build but at random points when i go in and out of full screen performance, fps and game speed will drop down by almost 50%. From 60 fps to around 30fps. I'll notice sometimes when going into fullscreen the mouse cursor won't auto hide. This is usally when it occurs or i'm able to reproduce it. Note this is with a 7700k and a gtx 1080ti so it isn't hardware related. I'm running puNES with the latest Windows 10 fall creators update. All of my drivers including the display driver are up to date. I've fully reinstalled the 64bit d3d9 WIP multiple times deleting every associated file to puNES. I don't know whats causing this. Happens with default settings and any settings it seems. Also I seem to be getting audio hitching even while the game appears to be running at fullspeed. I compared 4 games to Nestopia, Nintendulator, and MyNes. All sound fine, no hitching.

**Comment by WizLizard** [comment]
Ok did some more testing seems to only happen when my monitor is set to 60hz, when it's set to 144hz i get no audio hitching/popping. something isn't working correctly with a 60hz refresh.

**Comment by punesemu** [comment]
I tried two computers with windows 10 and I can not replicate the problem. I'll still try and keep you up to date.

**Comment by WizLizard** [comment]
So i was noticing weird performance issues with a good portion of my steam games that i wasn't having previously until I updated to the latest version of windows 10. Some games were plagued with strange stutter and lag. So bad it was making them unplayable. I did some google searches regarding Windows 10 fall update and stutter/bad performance. I came across a post in the Nvidia forums titled "Every game stuttering in Windows 10 fall update" or something to that effect. There was one suggested fix a guy recommend so i tried it. His recommendation was to enable windows game mode if it was disabled. Sure enough i had this disabled. As soon as i enabled it every game returned to it's buttery smooth performance. I haven't been able to reproduce this issue since.
